src/Features/Core/Portable/InitializeParameter/AbstractInitializeMemberFromParameterCodeRefactoringProviderMemberCreation.cs

```diff
@@ -52,7 +43,19 @@ internal abstract partial class AbstractInitializeMemberFromParameterCodeRefacto
                     ImmutableArray.Create(new SymbolSpecification.SymbolKindOrTypeKind(SymbolKind.Field))),
                     new NamingStyles.NamingStyle(Guid.NewGuid(), prefix: "_", capitalizationScheme: Capitalization.CamelCase),
                     enforcementLevel: ReportDiagnostic.Hidden));
+    }
```

Comment:
could you place in its own file.  generally we like one-type-per-file.  thanks1

Reply:
@wachulski Can you make this change as a second commit, and *not* amend the first commit to include it?

Reply:
@wachulski We accept a variety based on whatever works well for the individual contributor. In this case, I commented because the first commit is pretty easy to review without moving code to a new file, and then moving code in a separate commit will also be easy to review because the code itself wouldn't be changing.